client/sendProducts.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose')
+const Product = require("../server/models/Product");
+require('dotenv').config()
+
+mongoose.connect(process.env.DB_CONNECTION, {useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true})
+
+const productsArr = [
+    {
+        "name": "Pepperoni",
+        "image": "/files/pepperoni.jpg",
+        "price": "12BYN",
+        "description": "Pizza with pepperoni. No vegetarian",
+        "enabled": ["mozzarella", "pepperoni"]
+
+    },
+    {
+        "name": "4 cheese",
+        "image": "/files/4-cheese.jpg",
+        "price": "14BYN",
+        "description": "Cheesy pizza.Vegetarian",
+        "enabled": ["mozzarella", "blue cheese"]
+    },
+]
+
+const postPizza = () => {
+    productsArr.map((product) => {
+        const newProduct = new Product(product)
+        newProduct.save()
+      })
+};
+
+postPizza()
```

Comment:
@AStashevskaya Also good practice, it to notify, that script is finished, and wrap all script code in `try catch`, because it looks stucked, if nothing printed to console.

<img width="421" alt="image" src="https://user-images.githubusercontent.com/12561302/113599118-76c9af80-963e-11eb-9921-739545ba6b4c.png">